src/main/resources/application-cloud.yml

```diff
@@ -90,6 +90,25 @@ sample-svc:
     # time to read response
     read-timeout-milli-seconds: 5000
 
+action-svc:
+  action-plans-path: /actionplans
+  action-rules-path: /actionrules
+  action-rules-for-action-plan-path: /actionrules/actionplan/{actionPlanId}
```

Comment:
add the ID parameter onto the end of this and use the URI builder

Reply:
action-rule-path: /actionrules/{actipnRuleId} added